Lib/pyclbr.py

```diff
@@ -160,17 +160,20 @@ def _readmodule(module, path, inpackage=None):
     else:
         search_path = path + sys.path
     spec = importlib.util._find_spec_from_path(fullmodule, search_path)
+    if spec is None:
+        raise ModuleNotFoundError(f"no module named {fullmodule!r}", name=fullmodule)
```

Comment:
While this diff is the main fix I still think that `pyclbr._main` should try-except `readmodule_ex` and abort with a more fitting error message rather than dumping the exception traceback and quitting.

How viable would it be if this change could come along with the PR, if it were being addressed?

Reply:
I would prefer a separate PR if you want to start trying to clean up the `-m` operation of the script.